_pages/attributes.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: Available Attributes
+title: User attributes
 ---
 
-# Available attributes
+# User attributes
 
-Here are the available attributes that can be requested when a user is validated.
+Along with the user's UUID, here are the default attributes that can be requested when a user is validated.
 
-| Attribute      | Default |
-|----------------|---------|
-| first_name     | Yes     |
-| middle_name    | Yes     |
-| last_name      | Yes     |
-| address1       | Yes     |
-| address2       | Yes     |
-| city           | Yes     |
-| state          | Yes     |
-| zipcode        | Yes     |
-| dob            | Yes     |
-| ssn            | Yes     |
-| phone          | Yes     |
-| email          | No      |
+For LoA 1 accounts:
+
+- Email
+
+For LoA 3 accounts:
+
+- Email
+- First name
```

Comment:
Given that agency devs will use the dashboard to register their SP (and therefore don't need to know how the `yml` should be structured in the attribute bundle, is it still helpful to have it in the form you described? Maybe this is helpful to have in this format for the at-run-time request?